megamek/src/megamek/server/WeatherProcessor.java

```diff
@@ -113,7 +113,6 @@ private void resolveWeather() {
             }
         }
         if(((conditions.getWeather() == PlanetaryConditions.WE_HEAVY_SNOW)
-                || (conditions.getWeather() == PlanetaryConditions.WE_BLIZZARD)
```

Comment:
Where'd blizzard go?

Reply:
Global Warming? I just couldn't find it in the rules anymore.